app.js

```diff
@@ -0,0 +1,49 @@
+var createError = require('http-errors');
+var express = require('express');
+
+var path = require('path');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+const User = require("./models/user");
```

Comment:
You've created the User var for a reason.
But I can't find where you use it. Where? Or maybe it should be removed?